account_early_payment_discount/views/account_payment_term.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Form view -->
+    <record id="view_payment_term_form" model="ir.ui.view">
+        <field name="model">account.payment.term</field>
+        <field name="inherit_id" ref="account.view_payment_term_form"/>
+        <field name="arch" type="xml">
+            <!-- Boolean field for displaying other fields -->
+            <field name="active" position="after">
+                <field name="early_payment_discount"/>
+            </field>
+
+            <!-- Early Payment Discount configuration fields -->
+            <field name="note" position="after">
+                <group name="early_payment_discount_group" string="Early Payment Discount"
+                       attrs="{'invisible':[('early_payment_discount', '=', False)]}">
+
+                    <label for="epd_nb_days" />
+                    <div>
+                        <field name="epd_nb_days" class="oe_inline" attrs="{'required':[('early_payment_discount', '=', True)]}"/>
+                    </div>
+
+                    <label for="epd_discount" />
+                    <div>
+                        <field name="epd_discount" class="oe_inline" attrs="{'required':[('early_payment_discount', '=', True)]}"/>
+                        <label string="%" class="oe_inline"/>
+                    </div>
+
+                    <field name="company_currency" invisible="1" />
+                    <label for="epd_tolerance" />
+                    <div>
+                        <field name="epd_tolerance" class="oe_inline" widget='monetary' options="{'currency_field': 'company_currency'}"/>
+                    </div>
+                </group>
+            </field>
+        </field>
+    </record>
+
+    <!-- Tree view -->
+    <record model="ir.ui.view" id="view_payment_term_tree">
+        <field name="name">account.payment.term.tree</field>
+        <field name="model">account.payment.term</field>
```

Comment:
Why don't you inherit `account.view_payment_term_tree` ?

Reply:
I guess you mean https://github.com/odoo/odoo/blob/dccef8e41d45dd5150e4112aa3a2b19ba698dc62/addons/account/views/account_view.xml#L1631-L1640

I don't have the answer to your question: I have only migrated the module, not authored it :)
Then I don't know if there is a reason for replacing the view here.